style.css

```diff
@@ -0,0 +1,151 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+header {
+  padding: 80px 34px 82px 34px;
+  border-bottom-left-radius: 80px;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  background-image: url("./header-shapes\ mobile@2x.svg");
+  background-color: #e5e5e5;
+  position: relative;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+nav {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+  position: relative;
+  width: 100%;
+}
+
+nav:hover {
+  box-shadow: 0 0 12px 0 #6070ff;
+}
+
+nav > a {
+  font-size: 18px;
+  color: #6070ff;
+  line-height: 18px;
+  font-weight: 900;
+  text-decoration: none;
+}
+
+.humburger-btn {
+  display: flex;
+  flex-direction: column;
+  width: 40px;
+  height: 30px;
+  position: relative;
+}
+
+.span-1 {
+  position: absolute;
+  width: 100%;
+  height: 6px;
+  background-color: #6070ff;
+  margin-bottom: 3px;
+}
+
+.span-2 {
+  height: 6px;
+  position: absolute;
+  width: 100%;
+  background-color: #6070ff;
+  margin-top: 10px;
+}
+
+.span-3 {
+  height: 6px;
+  position: absolute;
+  width: 100%;
+  background-color: #6070ff;
+  margin-top: 20px;
+}
+
+hgroup > h1 {
+  font-size: 30px;
+  font-weight: 700;
+  color: #172bd4;
+  font-family: "Poppins", sans-serif;
+}
+
+hgroup > h2 {
+  color: #172bd4;
+  font-size: 25px;
+}
+
+.Iam {
+  color: #172bd4;
+}
+
+.Iam2 {
+  color: #6070ff;
+  font-weight: 700;
+}
+
+header > p {
+  font-size: 17px;
+  color: #04093b;
+  padding: 50px;
+  margin-bottom: 50px;
```

Comment:
- [ ] though you did a great job styling the `section` and `header`, it still needs some work to make it look like the template. Kindly style the `section` and `header` so that it looks similar to the design **i.e spacing (margin & padding), font, color, background color.**

Your Implementation | Figma Design
:---: | :---:
![image](https://user-images.githubusercontent.com/59416136/154277191-bec11bef-7f0c-4310-8158-e60b8723cc59.png) | ![image](https://user-images.githubusercontent.com/59416136/154277234-1610d60f-7995-4064-8d56-43581431fdf0.png)


Kindly fix it.

Reply:
Weldone @Daphineatim your design looks like the original

### Optional 
- Currently, there is  a [ scroll bar](https://www.w3schools.com/howto/howto_css_hide_scrollbars.asp) kindly fix it